site/layouts/partials/home_course_cards.html

```diff
@@ -26,7 +26,7 @@ <h3>New Courses</h3>
                                 {{ end }}
                                         <div class="item-wrapper {{ if not $isMobile }}col-{{ (div 12 $itemsInCarousel) }}{{ end }} w-100 d-flex justify-content-center">
                                             <div class="course-card card bg-white">
-                                                <a href="/courses/{{ $courseItem.url_path }}">
+                                                <a href="/courses/{{ $courseData.course_id }}">
```

Comment:
This should fix the home page but I wonder if the metadata should still be using `url_path` and not `course_id` to avoid breaking hugo when it will be parsing this metadata to publish the site. @gumaerc ?

Reply:
Note: this is regarding the future when hugo generates sites from these models, it won't break hugo now.

Reply:
actually nvm, it was already course_id, that didn't change.